HLTrigger/Configuration/python/HLT_75e33_timing_cff.py

```diff
@@ -136,10 +131,6 @@
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TkIsoEm22TkIsoEm12_cfi")
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TkIsoEm36_cfi")
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TripleTkMuon_5_3_3_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_BTV_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_Ele5_Open_L1Seeded_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_Ele5_Open_Unseeded_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_JME_cfi")
```

Comment:
are the removals of 

```
fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_BTV_cfi")
fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_JME_cfi")
```

intentional here?

Reply:
Ciao @mmusich the `MC_*` paths are not part of the `timing` version of the simplified menu. In #42759 those paths were removed from the `schedule`, so removing their `import` is a nice cleanup, IMHO.

Reply:
thanks for the clarification... still learning here :D